.github/workflows/ci.yml

```diff
@@ -0,0 +1,120 @@
+          #- "7.0"
+          #- "7.1"
+          #- "7.2"
+          - "7.3"
+    services:
+      app:
+        image: "glpi/githubactions-php:${{ matrix.php-version }}"
+        options: >-
+          --volume /glpi:/var/glpi
+      db:
+        image: "glpi/githubactions-${{ matrix.db-image }}"
+        env:
+          MYSQL_ALLOW_EMPTY_PASSWORD: "yes"
+        options: >-
+          --shm-size=1g
+      openldap:
+        image: "glpi/githubactions-openldap"
+    steps:
+      - name: "Checkout"
+        uses: "actions/checkout@v1"
+      - name: "Deploy source into app container"
+        run: |
+          sudo cp --no-target-directory --preserve --recursive `pwd` /glpi
+          sudo chown -R 1000:1000 /glpi
+      - name: "Initialize databases"
+        run: |
+          docker exec ${{ job.services.db.id }} mysql --user=root --execute="CREATE DATABASE \`glpi\`;"
+          docker exec ${{ job.services.db.id }} mysql --user=root --execute="CREATE DATABASE \`glpitest0723\`;"
+          cat tests/glpi-0.72.3-empty.sql | docker exec --interactive ${{ job.services.db.id }} mysql --user=root glpitest0723
+      - name: "Install dependencies"
+        run: |
+          docker exec ${{ job.services.app.id }} composer config --unset platform
+          docker exec ${{ job.services.app.id }} rm composer.lock
+          docker exec ${{ job.services.app.id }} composer --version
+          docker exec ${{ job.services.app.id }} composer install --optimize-autoloader --prefer-dist --no-interaction --no-progress --no-suggest
+      - name: "Install DB tests"
+        run: |
+          docker exec ${{ job.services.app.id }} bin/console glpi:database:install --config-dir=./tests --no-interaction --db-name=glpi --db-host=db --db-user=root
+          docker exec ${{ job.services.app.id }} bin/console glpi:database:update --config-dir=./tests --no-interaction | grep -q "No migration needed." || (echo "glpi:database:update command FAILED" && exit 1)
+      - name: "Unit tests"
+        run: |
+          docker exec ${{ job.services.app.id }} vendor/bin/atoum -p 'php -d memory_limit=512M' --debug --force-terminal --use-dot-report --configurations tests/telemetry.php --bootstrap-file tests/bootstrap.php --no-code-coverage -d tests/units
+      - name: "Functionnal tests"
+        run: |
+          docker exec ${{ job.services.app.id }} vendor/bin/atoum -p 'php -d memory_limit=512M' --debug --force-terminal --use-dot-report --configurations tests/telemetry.php --bootstrap-file tests/bootstrap.php --no-code-coverage --max-children-number 1 -d tests/functionnal
+      - name: "LDAP tests"
+        run: |
+          for f in `ls tests/LDAP/ldif/*.ldif`; do cat $f | docker exec --interactive ${{ job.services.openldap.id }} ldapadd -x -H ldap://127.0.0.1:3890/ -D "cn=Manager,dc=glpi,dc=org" -w insecure ; done
+          docker exec ${{ job.services.db.id }} mysql --user=root --execute="UPDATE \`glpi\`.\`glpi_authldaps\` SET \`host\` = 'openldap';"
```

Comment:
On Travis CI, LDAP server is reacheable on `127.0.0.1`, but here, we have to access it using its network alias `openldap`.
This SQL request may be removed if we drop Travic CI tests.